.circleci/config.yml

```diff
@@ -339,17 +333,16 @@ workflows:
       - py37-trinity-integration
       - py37-beacon
 
-      - py36-native-state-byzantium
-      - py36-native-state-frontier
-      - py36-native-state-homestead
-      - py36-native-state-eip150
-      - py36-native-state-eip158
```

Comment:
These are still state tests, they just happen to be in a different folder. I think the job name could stay the same.

Reply:
They're blockchain tests now! Blockchain tests which happen to also contain state tests. The command for this specific one is now: 

> native-blockchain-eip158: pytest {posargs:tests/json-fixtures/test_blockchain.py --fork EIP158}

Reply:
Somewhat orthogonal but it would be good to rename the `EIPXXX` to their long form fork names.

Reply:
What are their long form fork names? The EIP's page just [calls it 158](https://eips.ethereum.org/EIPS/eip-158) and doesn't say when it was scheduled.

Reply:
150 is apparently Tangerine Whistle and yeah, that name is 200% more exciting than EIP-150 :grinning:

Reply:
Do you want me to do that in this PR?

Reply:
I think "Spurious Dragon" is the other one.  Look in `eth.vm.forks` and you can find them (and there should be things within each module to hint at which EIPXXX they correspond to.